git_commands.txt

```diff
@@ -0,0 +1,5 @@
+git init
+git clone
+git config
+git status
+
```

Comment:
Did you learn only these commands?
Explanation of the command usage is missing. Check with Phani team you will know what I'm taking.